source/includes/_ethereum.md

```diff
@@ -792,5 +1087,33 @@ contract proofShieldExample is usingOraclize {
 }
 ```
 
+### Byte Datasources
+
+As a more advanced feature, Provable are providing all datasource `strings` as type `bytes1` as well as `strings`. The strings remain more readable (`URL`, `Random` and so on…) in Provable queries, but as with any string-related manipulation in Solidity, using them costs more gas-wise.  As such, by offering the same set of datasources as a `bytes1` equivalent, advanced users may choose to declare their datasource in this manner instead and thus benefit from cheaper query gas-costs.
+
+<aside class="notice">
+All of Provable's API functions where a `string _datasource` parameter is used can have the parameter substituted for its `bytes1` equivalent without altering the behaviour of the function in any way. See the section on the right for the `string -> bytes1` datasource conversion dictionary.
+</aside>
+
+``` javascript
+
+// Datasource strings and their `bytes1` equivalents:
+
+|---------------------------|
+|       `string` | `bytes1` |
+|---------------------------|
```

Comment:
lel...

I'm fine with it as is, but it would make more sense to see them in desc/asc bytes1 order.

Reply:
Common-sense beats pyramid, so done.